franka_example_controllers/src/cartesian_impedance_example_controller.cpp

```diff
@@ -0,0 +1,251 @@
+#include "franka_example_controllers/cartesian_impedance_example_controller.hpp"
+#include "franka_example_controllers/pseudo_inversion.hpp"
+
+const Eigen::MatrixXd IDENTITY = Eigen::MatrixXd::Identity(6, 6);
```

Comment:
Are you using this somewhere? I think you always use directly the `Identity` function whereever you needed it..?

Reply:
you're right, it was an old overlap